services/themeConfig.js

```diff
@@ -13,9 +15,11 @@ const queryBuilder     = new QueryBuilder(ThemeConfig, restrictedFields, default
 const getThemeConfig = async (PostBody) => {
     if (!PostBody) throw NSErrors.PostBodyUndefined;
 
-    const config    = await Configuration.findOne({});
-    PostBody.filter = {name: config.environment.currentTheme};
-    return queryBuilder.findOne(PostBody);
+    const config        = await Configuration.findOne({});
+    PostBody.filter     = {name: config.environment.currentTheme};
+    const filenames     = fs.readdirSync(path.join(global.appRoot, 'themes/default_theme/demoDatas'));
```

Comment:
The "sync" methods in fs are prone to errors, you should use utils/fsp, they use callback in promise style to ensure the end of the system request
```js
const filenames = await fs.readdir(path.join(global.appRoot, 'themes/default_theme/demoDatas'));
```